server/src/infra/entities/exif.entity.ts

```diff
@@ -43,6 +54,15 @@ export class ExifEntity {
   @Column({ type: 'float', nullable: true })
   longitude!: number | null;
 
+  @Column({
+    type: 'enum',
+    enum: ProjectionType,
+    enumName: 'projectionType',
+    default: ProjectionType.NONE,
+    nullable: true,
+  })
+  projectionType!: ProjectionType | null;
```

Comment:
`ProjectionType` has `NONE`, so we don't need `null`

Reply:
fixed